client/src/useFileManager.js

```diff
@@ -81,6 +91,7 @@ export default function useFileManager() {
         sessionRef.current = {
           url: responseBody.url,
           dir: responseBody.dir,
+          home_dir: responseBody.home_dir,
```

Comment:
JavaScript likes camelCase identifiers.  The key should be renamed to `homeDir` please.  I'm happy to have the JSON as either `home_dir` or `homeDir`.